templates/components/faceted-search/show-more-facets.html

```diff
@@ -9,12 +9,21 @@ <h1 class="facet-quick-heading">
         {{#each facets}}
             {{#if facet '===' ../all_facets}}
                 {{#each items}}
-                    <li class="navList-item"><a
+                    <li class="navList-item">
+                        <a
                         href="{{url}}"
                         class="navList-action {{#if selected }} is-active {{/if}}"
                         rel="nofollow"
                         data-id="{{ id }}"
-                        data-faceted-search-facet>{{title}} ({{count}})</a></li>
+                        data-faceted-search-facet>
+                        {{title}}
+                        {{#if ../show_product_counts}}
+                            {{#if count}}
+                            <span>({{count}})</span>
+                            {{/if}}
+                        {{/if}}
```

Comment:
Since you brought the `a` tag down, give this  block an indentation too, it'll make it easier for the next person to read a bit.

So it'll look something like:
```
                    <li class="navList-item">
                        <a
                            href="{{url}}"
                            class="navList-action {{#if selected }} is-active {{/if}}"
                            rel="nofollow"
                            data-id="{{ id }}"
                            data-faceted-search-facet
                        >
                            {{title}}
                            {{#if ../show_product_counts}}
                                {{#if count}}
                                    <span>({{count}})</span>
                                {{/if}}
                            {{/if}}
                        </a>
                    </li>
```